components/main_layout/public/icons/bid-status-arrow.svg

```diff
@@ -0,0 +1,4 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 8.14 7.05">
```

Comment:
It's would be good to move this to `components/bid_status/public/icons/bid-status-arrow.svg` to co-located this asset.